src/components/organisms/top/top.scss

```
@@ -0,0 +1,50 @@
+@use "../../variables" as *;
+
+.top {
+  .top-wrap {
+    position: relative;
+
+    img {
+      width: 100%;
+      height: 100vh;
+      object-fit: cover;
+      @media screen and (max-width: $breakpoint-lg) {
+        height: 360px;
+      }
+    }
+    .front-message {
+      padding: 0;
+      display: flex;
+      align-items: left;
+
+      h2 {
+        font-family: $fontfamily-mincho;
+        font-size: $fontsize-xl;
+        line-height: 46px;
+        margin: 0;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        color: $white;
+        writing-mode: vertical-rl;
+
+        @media screen and(max-width:$breakpoint-lg) {
+          line-height: 30px;
+          font-size: $fontsize-l;
+        }
+      }
+    }
+  }
+  .message {
```

paddingについてですが、.advertisementに対して指定するとうまくいくと思います。
値については確認お願いします
```suggestion
.advertisement {
  background-color: $black;
  padding: md, lgでそれぞれ別のやつつける
  .message {
```